tests/test_asyncio/test_search.py

```diff
@@ -1560,3 +1568,53 @@ async def test_query_timeout(decoded_r: redis.Redis):
     q2 = Query("foo").timeout("not_a_number")
     with pytest.raises(redis.ResponseError):
         await decoded_r.ft().search(q2)
+
+
+@pytest.mark.redismod
+@skip_if_resp_version(3)
```

Comment:
Can't we run the test for RESP3 also?

Reply:
We can't because we just return plain dict on RESP3 without any decoding and parsing result.

Reply:
We can assert differently depending on RESP2/RESP3, there are already tests doing that.

Reply:
I think I actually get it, it's because of the vector data...